hbase-server/src/test/java/org/apache/hadoop/hbase/security/access/TestSnapshotScannerHDFSAclController.java

```diff
@@ -158,6 +158,7 @@ public void testGrantGlobal1() throws Exception {
 
     TestHDFSAclHelper.createTableAndPut(TEST_UTIL, table);
     snapshotAndWait(snapshot1, table);
+    snapshotAndWait(snapshot2, table);
```

Comment:
Does this test really need this change, or is this part of the PR by mistake?

Reply:
it actually does, i am still trying to understand why the order of granting permissions to global user matter in the order. e.g. before table creation, granting global user permission allows user to scan snapshot but after table creation if we grant it, it doesn't work.
perhaps some bug in coproc `SnapshotScannerHDFSAclController`

Reply:
Ack!